src/state/transactions/updater.tsx

```diff
@@ -75,6 +76,8 @@ export default function Updater() {
       sendAnalyticsEvent(SwapEventName.SWAP_TRANSACTION_COMPLETED, {
         // if timeToSwap was already set, we already logged this session
         time_to_swap: hasReportedTimeToSwap ? undefined : elapsedTime,
+        time_to_sign: hasReportedTimeToSwap ? undefined : SignedTransactionTimestampRegistry.getInstance().get(hash),
```

Comment:
i dot think this is right, this measures the amount of time it takes post sign -> tx is successful, whereas your time to sign should just be pre-signature -> post-signature